Languages/IronPython/IronPython.Modules/hashlib/_md5.cs

```diff
@@ -0,0 +1,102 @@
+/* ****************************************************************************
+ *
+ * Copyright (c) Microsoft Corporation. 
+ *
+ * This source code is subject to terms and conditions of the Apache License, Version 2.0. A 
+ * copy of the license can be found in the License.html file at the root of this distribution. If 
+ * you cannot locate the  Apache License, Version 2.0, please send an email to 
+ * ironpy@microsoft.com. By using this source code in any fashion, you are agreeing to be bound 
+ * by the terms of the Apache License, Version 2.0.
+ *
+ * You must not remove this notice, or any other, from this software.
+ *
+ *
+ * ***************************************************************************/
+
+#if FEATURE_FULL_CRYPTO // MD5
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Scripting.Runtime;
+using System.Reflection;
+using System.Security.Cryptography;
+using System.Text;
+using IronPython.Runtime;
+using IronPython.Runtime.Operations;
+
+//!!! This is pretty inefficient. We should probably use hasher.TransformBlock instead of
```

Comment:
Can we get rid of these comments (since it looks like that's exactly what you did). Same comment for `_sha.cs`, `_sha256.cs` and `_sha512.cs`.